src/server.ts

```diff
@@ -2,10 +2,18 @@ import express from "express";
 import { Request, Response } from "express";
 import { config } from "./config";
 import { logger } from "./logger";
-import { noAppPortMsg, noEnvMsg, noApiKeyMsg, defaultScore } from "./constants";
+import {
+  noAppPortMsg,
+  noEnvMsg,
+  noApiKeyMsg,
+  defaultScore,
+  DB_URL,
+} from "./constants";
 import OmdbService from "./services/omdbService";
 import MovieStore from "./stores/movieStore";
 
+const MongoClient = require("mongodb").MongoClient;
```

Comment:
You can get benefits from typescript, if you do import properly.
```
import * as mongoDB from "mongodb";
const MongoClient = mongoDB.MongoClient;
```
Now you have autocomplete and correct return types.